From d05c26ce690e867aabfc7d708d481e0f86f23496 Mon Sep 17 00:00:00 2001
From: Eilon Greenstein <eilong@broadcom.com>
Date: Sat, 17 Jan 2009 23:26:13 -0800
Subject: bnx2x: Version update
Acked-by: Karsten Keil <kkeil@novell.com>
Reference: bnc#472500

Updating the version and the year of updated files

Signed-off-by: Eilon Greenstein <eilong@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/bnx2x.h      |    2 +-
 drivers/net/bnx2x_link.c |    2 +-
 drivers/net/bnx2x_main.c |    6 +++---
 drivers/net/bnx2x_reg.h  |    2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

Index: linux-2.6.27-bnx2x_2/drivers/net/bnx2x.h
===================================================================
--- linux-2.6.27-bnx2x_2.orig/drivers/net/bnx2x.h
+++ linux-2.6.27-bnx2x_2/drivers/net/bnx2x.h
@@ -1,6 +1,6 @@
 /* bnx2x.h: Broadcom Everest network driver.
  *
- * Copyright (c) 2007-2008 Broadcom Corporation
+ * Copyright (c) 2007-2009 Broadcom Corporation
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
Index: linux-2.6.27-bnx2x_2/drivers/net/bnx2x_link.c
===================================================================
--- linux-2.6.27-bnx2x_2.orig/drivers/net/bnx2x_link.c
+++ linux-2.6.27-bnx2x_2/drivers/net/bnx2x_link.c
@@ -1,4 +1,4 @@
-/* Copyright 2008 Broadcom Corporation
+/* Copyright 2008-2009 Broadcom Corporation
  *
  * Unless you and Broadcom execute a separate written software license
  * agreement governing use of this software, this software is licensed to you
Index: linux-2.6.27-bnx2x_2/drivers/net/bnx2x_main.c
===================================================================
--- linux-2.6.27-bnx2x_2.orig/drivers/net/bnx2x_main.c
+++ linux-2.6.27-bnx2x_2/drivers/net/bnx2x_main.c
@@ -1,6 +1,6 @@
 /* bnx2x_main.c: Broadcom Everest network driver.
  *
- * Copyright (c) 2007-2008 Broadcom Corporation
+ * Copyright (c) 2007-2009 Broadcom Corporation
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -57,8 +57,8 @@
 #include "bnx2x.h"
 #include "bnx2x_init.h"
 
-#define DRV_MODULE_VERSION	"1.45.23"
-#define DRV_MODULE_RELDATE	"2008/11/03"
+#define DRV_MODULE_VERSION	"1.45.24"
+#define DRV_MODULE_RELDATE	"2009/01/14"
 #define BNX2X_BC_VER		0x040200
 
 /* Time in jiffies before concluding the transmitter is hung */
Index: linux-2.6.27-bnx2x_2/drivers/net/bnx2x_reg.h
===================================================================
--- linux-2.6.27-bnx2x_2.orig/drivers/net/bnx2x_reg.h
+++ linux-2.6.27-bnx2x_2/drivers/net/bnx2x_reg.h
@@ -1,6 +1,6 @@
 /* bnx2x_reg.h: Broadcom Everest network driver.
  *
- * Copyright (c) 2007-2008 Broadcom Corporation
+ * Copyright (c) 2007-2009 Broadcom Corporation
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
